requests/request.go

```diff
@@ -275,3 +276,59 @@ func (request *Request) File(path string, params FileUploadParams, extraHeaders
 
 	return request.doRequestResponse(req)
 }
+
+var urlMap = []string{
+	constants.ORDER_URL,
+	constants.INVOICE_URL,
+	constants.PAYMENT_URL,
+	constants.PaymentLink_URL,
+	constants.REFUND_URL,
+	constants.CARD_URL,
+	constants.CUSTOMER_URL,
+	constants.ADDON_URL,
+	constants.TRANSFER_URL,
+	constants.VIRTUAL_ACCOUNT_URL,
+	constants.SUBSCRIPTION_URL,
+	constants.PLAN_URL,
+	constants.QRCODE_URL,
+	constants.FUND_ACCOUNT_URL,
+	constants.SETTLEMENT_URL,
+	constants.ITEM_URL,
+	constants.METHODS_URL,
+	constants.ACCOUNT_URL,
+	constants.STAKEHOLDER_URL,
+	constants.PRODUCT_URL,
+	constants.TNC,
+	constants.IIN,
+	constants.WEBHOOK,
+}
+
+func SearchInSlice(needle string) bool {
+
+	for _, b := range urlMap {
+		if b == needle {
+			return true
```

Comment:
we can do this to optimize it from o(n) to o(1)

var urlMap = map[string]struct{}{
	constants.ORDER_URL:          {},
	constants.INVOICE_URL:        {},
	constants.PAYMENT_URL:        {},
	constants.PaymentLink_URL:    {},
	constants.REFUND_URL:         {},
	constants.CARD_URL:           {},
	constants.CUSTOMER_URL:       {},
	constants.ADDON_URL:          {},
	constants.TRANSFER_URL:       {},
	constants.VIRTUAL_ACCOUNT_URL: {},
	constants.SUBSCRIPTION_URL:   {},
	constants.PLAN_URL:           {},
	constants.QRCODE_URL:         {},
	constants.FUND_ACCOUNT_URL:   {},
	constants.SETTLEMENT_URL:     {},
	constants.ITEM_URL:           {},
	constants.METHODS_URL:        {},
	constants.ACCOUNT_URL:        {},
	constants.STAKEHOLDER_URL:    {},
	constants.PRODUCT_URL:        {},
	constants.TNC:                {},
	constants.IIN:                {},
	constants.WEBHOOK:            {},
}

func SearchInMap(needle string) bool {
	_, found := urlMap[needle]
	return found
}